### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.4.7 - style easily perceivable focus states for all interactive elements

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1305]
# Feature Request

## Description
For keyboard navigation, the position of keyboard focus must be visible and easily distinguishable on any interactive element. 
Browsers make focus visible by default, but on dark background or linked images focus is not visible enough. Also, some elements have a costum style focus that does not meet minimum contrast requirements (form fields, buttons, button-stlye links, icon links in footer, etc.). 

## Solution
Provide one consistend styling for all focusable elements; a double border in black and white, 2px each, is useful, because its visible on all backgrounds. 
Where necessary, make sure the focus border does not interfere with readability of link text. 
In some cases, a gap is needed between the border and the element itself (see screenshots)
However, the border should not be obscured by other elements, as is currently the case with icon links in the footer. 

## context, links and screenshots
![image](https://github.com/benjaminkott/bootstrap_package/assets/132338236/69f1430b-5509-40b7-887a-73748af02c11)
focus not visible enough

![image](https://github.com/benjaminkott/bootstrap_package/assets/132338236/bf35de53-90a6-4fd7-9ebf-709fdf9ac28f)
focus with extra padding

![image](https://github.com/benjaminkott/bootstrap_package/assets/132338236/88142462-bbb3-440b-993c-7f0ba08f5d58)
partially obscured focus box

**Comment by benjaminkott** [comment]
Imagelinks adjusted:
![image](https://github.com/benjaminkott/bootstrap_package/assets/3243119/04240480-9724-44bb-9541-1876d9667858)

Focus not cut off anymore on social icons
![image](https://github.com/benjaminkott/bootstrap_package/assets/3243119/e8219dd4-3799-4cb4-a290-133f138e3757)

**Comment by SarahEdding** [comment]
fixed for all instances except carousel slides. I've openes a seperate issue for that: https://github.com/benjaminkott/bootstrap_package/issues/1383
